doc/getting-started/apl-nuc.rst

```diff
@@ -314,6 +304,8 @@ Set up Reference UOS
    .. code-block:: none
 
       # cd ~
+      # mkdir uos
```

Comment:
since you changed the commands in the previous section to not be running as root, should all these prompts be a ``$`` now too?